src/NHibernate/Driver/DriverBase.cs

```diff
@@ -338,7 +338,7 @@ public DbParameter GenerateOutputParameter(DbCommand command)
 
 		public virtual bool RequiresTimeSpanForTime => false;
 
-#if NETCOREAPP2_0_OR_GREATER
+#if NETCOREAPP2_0_OR_GREATER || NETSTANDARD2_1_OR_GREATER
 		public virtual bool SupportsSystemTransactions => false;
 
 		public virtual bool SupportsNullEnlistment => false;
```

Comment:
Shouldn't we invert the condition?
```
#if NETFX
		public virtual bool SupportsSystemTransactions => true;

		public virtual bool SupportsNullEnlistment => true;
#else
		public virtual bool SupportsSystemTransactions => false;

		public virtual bool SupportsNullEnlistment => false;
#endif
```

Reply:
Previously, for some reason, for .NET Standard 2.0 it was the same as for .NET Fx.

Reply:
@fredericDelaporte done

Reply:
That was coming from [here](https://github.com/nhibernate/nhibernate-core/pull/1523/files#r160946816). Theoretically, System.Transactions are supported in all our targets. But at the time, only NET Fx data providers were actually supporting them. (Even non-distributed ambient transaction were actually supported only by NET Fx data providers.)

As these properties are only used for tests, and do not change the behavior of the NHibernate library, setting them to false for the appcore target was the way to avoid testing these unsupported ambient transactions under that target. I do not think there was much thoughts about whether it should be set to true or false for netstandard.